pcf/cli/commands/__init__.py

```diff
@@ -0,0 +1,31 @@
+import click
+
+COMMON_OPTIONS = [
```

Comment:
these are just common options for the apply commands. so maybe change to apply_options or common_apply_options.